client/rest-high-level/src/test/java/org/elasticsearch/client/CrudIT.java

```diff
@@ -64,10 +63,7 @@ public void testExists() throws IOException {
         }
         {
             GetRequest getRequest = new GetRequest("index", "type", "does_not_exist").version(1);
-            ElasticsearchException exception = expectThrows(ElasticsearchException.class,
-                    () -> execute(getRequest, highLevelClient()::exists, highLevelClient()::existsAsync));
-            assertEquals(RestStatus.BAD_REQUEST, exception.status());
-            assertThat(exception.getMessage(), containsString("/index/type/does_not_exist?version=1: HTTP/1.1 400 Bad Request"));
+            assertFalse(execute(getRequest, highLevelClient()::exists, highLevelClient()::existsAsync));
```

Comment:
Should we consider this a breaking change? The Exists response is changed with #23186